src/api/api.h

```diff
@@ -0,0 +1,12 @@
+#ifndef OIDC_AGENT_API_H
+#define OIDC_AGENT_API_H
+
+#include "error.h"
+#include "memory.h"
+#include "tokens.h"
+#ifdef __MSYS__
+#include "accounts.h"
```

Comment:
Currently I have the accounts functions only in the windows version of the api. In the future we can also add them to the other versions.